src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/Dashboard.stories.tsx

```diff
@@ -155,6 +157,12 @@ const DashboardWrapper = (props: DashboardWrapperProps) => {
     createRandomScanResult({ status: "removed", manually_resolved: true }),
   ];
 
+  const mockedAnnouncements: UserAnnouncementWithDetails[] = [
+    createRandomAnnouncement(),
+    createRandomAnnouncement(),
+    createRandomAnnouncement(),
+  ];
```

Comment:
Could remove the repetition here if you want:

```suggestion
const mockedAnnouncements: UserAnnouncementWithDetails[] = Array.from({ length: 3 }, createRandomAnnouncement);
```